source/retryable-writes/retryable-writes.rst

```diff
@@ -242,8 +237,8 @@ These methods above are defined in the `CRUD`_ specification.
 
 Later versions of MongoDB may add support for additional write operations.
 
-Unsupported Write Operations (client-side error)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Unsupported Write Operations
```

Comment:
This section does a great job of telling you what write operations aren't supported, and therefore which to not add a transaction id to, but it never says not to retry these write operations.

Reply:
https://github.com/mongodb/specifications/pull/174/commits/25623b6dcd077edae6704e12d526cbb5c397d6d6 adds clarification.